apps/arena/lib/arena/game/effect.ex

```diff
@@ -229,8 +229,11 @@ defmodule Arena.Game.Effect do
 
         send(self(), {:damage_done, pool_owner.id, real_damage})
 
-        Entities.take_damage(entity, real_damage)
-        |> maybe_send_to_killfeed(pool_owner.id)
+        if Entities.alive?(entity) do
+          Entities.take_damage(entity, real_damage, pool_owner.id)
```

Comment:
Do you think it makes sense to move `Entities.alive?/1` check to be inside of `Entities.take_damage/3`?

Reply:
it totally does! i just did it